packages/bpk-tokens/package.json

```diff
@@ -11,5 +11,8 @@
     "url": "git@github.com:Skyscanner/backpack.git"
   },
   "author": "Backpack Design System <backpack@skyscanner.net>",
-  "license": "Apache-2.0"
+  "license": "Apache-2.0",
+  "devDependencies": {
```

Comment:
This should be a `"dependency"` - this will fail otherwise when a consumer `npm install`'s bpk-tokens as only `"dependency"`'s are installed.